### ISSUE TITLE
Whereiscolotoday enhancement

### ISSUE THREAD

**Issue by Jashaul** [akashsara/homebrew_helper#45]
To add a command to find the region colo woke up. 

PS. exclusive permission to allow colo to edit

**Comment by akashsara** [comment]
Please give a better description of how this should work.

**Comment by akashsara** [comment]
Maybe:

!whereiscolotoday
> Location in format [City], [Country] where the current time is 10 AM. Randomly choose between cities if there are multiple.

**Comment by Jashaul** [comment]
Yeah, there should be an api available for that. We could also do it from scratch, seems simple to implement.

custom logic:
- get the current UTC time
- match with all time zones currently at 10AM
- fetch a list of cities/countries under that timezone and pick one at random
